src/main/res/values/strings.xml

```diff
@@ -3,7 +3,7 @@
 
     <string name="app_name">Syncthing</string>
 
-    <string name="app_description">Replace proprietary services with something open, trustworthy and decentralized.</string>
+    <string name="app_description">An open, trustworthy and decentralized file synchronization application.</string>
```

Comment:
I took this from [syncthing.net](https://syncthing.net/), do you really think we should have a different text?